request-management-api/request_api/models/masterdataimport/programareas.sql

```diff
@@ -1,35 +1,34 @@
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Citizens’ Services ','BC GOV Ministry', True,'CITZ ','CTZ');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Education ','BC GOV Ministry', True,'EDUC ','EDU');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Energy, Mines and Low Carbon Innovation ','BC GOV Ministry', True,'EMLI ','EML');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Environment and Climate Change Strategy ','BC GOV Ministry', True,'ENV ','MOE');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Finance ','BC GOV Ministry', True,'FIN ','FIN');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Forests, Lands, Natural Resource Operations and Rural Development ','BC GOV Ministry', True,'FLNR ','FNR');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Health ','BC GOV Ministry', True,'HLTH ','HTH');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Indigenous Relations and Reconciliation ','BC GOV Ministry', True,'IRR ','IRR');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Jobs, Economic Recovery and Innovation ','BC GOV Ministry', True,'JERI ','JER');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Labour ','BC GOV Ministry', True,'LBR ','LBR');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Mental Health and Addictions ','BC GOV Ministry', True,'MMHA ','MHA');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Municipal Affairs ','BC GOV Ministry', True,'MUNI ','MMA');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Public Safety and Solicitor General ','BC GOV Ministry', True,'PSSG ','PSS');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Social Development and Poverty Reduction ','BC GOV Ministry', True,'SDPR ','MSD');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Tourism, Arts, Culture and Sport ','BC GOV Ministry', True,'TACS ','TAC');
-INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Transportation and Infrastructure ','BC GOV Ministry', True,'TRAN ','TRA');
+INSERT INTO public."ProgramAreas"(name, type, isactive, bcgovcode, iaocode) VALUES ('Ministry of Citizens’ Services ','BC GOV Ministry', True,'CITZ','CTZ');
```

Comment:
I'm a bit alarmed by us changing these fixed SQL files.  I know we're just fixing typos here, but there's no way for us to just update the dev/test/prod DBs with these changes, it requires us purging all data and manually running this again which isn't great.

Instead, we should look at Services BC queue-management API.  They have all this starter data defined in python, so it automatically is run with migrations.  You can also set different data for diff environments.  Example: https://github.com/bcgov/queue-management/blob/master/api/config.py

Reply:
yes. this is for our initial setup. We can discuss this in out technical meeting